src/GafferImage/VectorWarp.cpp

```diff
@@ -51,17 +51,17 @@ using namespace GafferImage;
 // Engine implementation
 //////////////////////////////////////////////////////////////////////////
 
-struct UVWarp::Engine : public Warp::Engine
+struct VectorWarp::Engine : public Warp::Engine
 {
 
-	Engine( const Box2i &displayWindow, const Box2i &tileBound, const Box2i &validTileBound, ConstFloatVectorDataPtr uData, ConstFloatVectorDataPtr vData, ConstFloatVectorDataPtr aData, VectorMode vectorMode, VectorUnits vectorUnits )
+	Engine( const Box2i &displayWindow, const Box2i &tileBound, const Box2i &validTileBound, ConstFloatVectorDataPtr xData, ConstFloatVectorDataPtr yData, ConstFloatVectorDataPtr aData, VectorMode vectorMode, VectorUnits vectorUnits )
 		:	m_displayWindow( displayWindow ),
 			m_tileBound( tileBound ),
-			m_uData( uData ),
-			m_vData( vData ),
+			m_xData( xData ),
+			m_yData( yData ),
 			m_aData( aData ),
-			m_u( uData->readable() ),
-			m_v( vData->readable() ),
+			m_u( xData->readable() ),
```

Comment:
Any reason not to rename `m_u` and `m_v` as well? It would make sense to me, especially since they're paired with the `m_*Data` variables which have been renamed.